composer/loggers/wandb_logger.py

```diff
@@ -98,11 +99,16 @@ def __init__(
     def _set_is_in_atexit(self):
         self._is_in_atexit = True
 
-    def log_data(self, state: State, log_level: LogLevel, data: Dict[str, Any]):
+    def log_hyperparameters(self, hyperparameters: Dict[str, Any]):
         import wandb
-        del log_level  # unused
-        if self._enabled:
-            wandb.log(data, step=int(state.timestamp.batch))
+        wandb.config.update(hyperparameters)
+
+    def log_metrics(self, metrics: Dict[str, float], step: Optional[int] = None) -> None:
+        import wandb
```

Comment:
Why are these `import wandb` in line?

Reply:
because it doesn't get imported anywhere else. We don't want it as a top-level import because the user may not have it installed